crates/file/src/blob.rs

```diff
@@ -0,0 +1,44 @@
+pub use crate::file::File;
+
+pub trait Blob: raw::Blob {
+    fn size(&self) -> usize {
+        self.raw().size() as usize
+    }
+
+    fn mime_type(&self) -> Option<mime::Mime> {
+        Some(self.raw().type_().parse().ok()?)
```

Comment:
Have you looked at the difference in code size between returning the raw string vs using the `mime` crate to parse? If it is sizable, we probably want to different methods, and maybe a feature for `mime`